src/altinn-app-frontend/src/components/message/ErrorReport.tsx

```diff
@@ -80,22 +89,74 @@ const ErrorReport = ({ components }: IErrorReportProps) => {
         return;
       }
       ev.preventDefault();
-      if (currentView === error.layout) {
-        dispatch(
-          FormLayoutActions.updateFocus({
-            focusComponentId: error.componentId,
-          }),
-        );
-      } else {
+      if (currentView !== error.layout) {
         dispatch(
           FormLayoutActions.updateCurrentView({
             newView: error.layout,
             runValidations: null,
             returnToView: currentView,
-            focusComponentId: error.componentId,
           }),
         );
       }
+
+      const nodes = nodesInLayout(layouts[error.layout], repeatingGroups);
+      const componentNode = nodes.findById(error.componentId);
+
+      // Iterate over parent repeating groups
+      componentNode.parents().forEach((parent, i, allParents) => {
+        if (
+          parent.item?.type == 'Group' &&
+          parent.item.edit?.mode !== 'likert' &&
+          parent.item.maxCount > 1
+        ) {
+          const child =
+            i == 0
+              ? componentNode
+              : (allParents[i - 1] as AnyChildNode<'unresolved'>);
+
+          // Go to correct multiPage page if necessary
+          if (
+            (parent.item as RepeatingGroupHierarchy<'unresolved'>).edit
```

Comment:
If you had assigned `parent.item` to a variable before the `if` above, I think TypeScript might have inferred this type automatically